core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala

```diff
@@ -138,12 +138,13 @@ trait WorkspaceSupport {
   def withWorkspaceBucketRegionCheck[T](bucketRegion: Option[String])(op: => Future[T]): Future[T] = {
     bucketRegion match {
       case Some(region) =>
-        // if the user specifies a region for the workspace bucket, it must be in the proper format for a single region
+        // if the user specifies a region for the workspace bucket, it must be in the proper format for a single region or the default bucket location (US multi region)
         val singleRegionPattern = "[A-Za-z]+-[A-Za-z]+[0-9]+"
-        if (region.matches(singleRegionPattern)) op
+        val defaultBucketRegion = "US"
```

Comment:
It took me a second to actually wrap my head around what we're doing here.  To me, it doesn't matter that this is the `defaultBucketRegion` so much as the acceptable patterns for the bucket region are `[A-Za-z]+-[A-Za-z]+[0-9]+` OR `US`.  

So it might read clearer if we just updated the regex pattern to be something like `US$|[A-Za-z]+-[A-Za-z]+[0-9]+` or just rename this variable, something like:

```suggestion
        val validUSPattern = "US"
```

I'm also considering whether rewording line 147 might might help or not.

Anyway, given where we are, this is not enough to make me block the PR, but I do think these changes might make this clearer in the future.